PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionExDialog.java

```diff
@@ -55,6 +55,7 @@
     public SparkSubmissionExDialog(@NotNull Project project, @NotNull List<IClusterDetail> cachedClusterDetails, @Nullable CallBack callBack) {
         this.project = project;
         this.callBack = callBack;
+
```

Comment:
What does it mean?

---
In reply to: [117161406](https://github.com/Microsoft/azure-tools-for-java/pull/459#discussion_r117161406) [](ancestors = 117161406)